project_task_materials/models/project_task.py

```diff
@@ -47,10 +47,10 @@ def _get_stock_status(self):
             procurement_ids = self.env['procurement.order'].search(
                 [('material_project_task_id.id', '=', item.id)])
             for line in procurement_ids:
-                operation_move_link = stock_move_operation_link.search(
+                operation_move_link_ids = stock_move_operation_link.search(
                     [('move_id.procurement_id', '=', line.id)])
-                qty_done = operation_move_link.operation_id.qty_done
-                item.qty_delivered = qty_done
+                for x in operation_move_link_ids:
+                    item.qty_delivered += x.operation_id.qty_done
```

Comment:
qty_done = 0
for x in operation_move_link_ids:
  qty_done += x.operation_id.qty_done
item.qty_delivered = qty_done